clients/cli/src/clap_app.rs

```diff
@@ -1721,6 +1722,47 @@ pub fn app<'a>(
                 .nonce_args(true)
                 .offline_args(),
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::UnwrapLamports.into())
+                .about("Unwrap lamports from a SOL token account")
+                .arg(
+                    Arg::with_name("amount")
+                        .value_parser(Amount::parse)
+                        .value_name("TOKEN_AMOUNT")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("Amount to unwrap, in tokens; accepts keyword ALL"),
```

Comment:
I'm torn about this -- the command is `UnwrapLamports`, which implies we should pass in a lamport amount, but the CLI uses UI amounts almost everywhere (ie SOL).

Maybe we should just be super clear:
```suggestion
                    Arg::with_name("amount")
                        .value_parser(Amount::parse)
                        .value_name("TOKEN_AMOUNT")
                        .takes_value(true)
                        .index(1)
                        .required(true)
                        .help("Amount to unwrap, in lamports; accepts keyword ALL"),
```